### ISSUE TITLE
BigQuery Integration

### ISSUE THREAD

**Issue by lihan** [pacuna/snowplow-pipeline#1]
Hi @pacuna , any idea how to get those events from kafka into BigQuery?

**Comment by pacuna** [comment]
HI @lihan.

The easiest way would be to add the BigQuery library to the `events-processor` app in the repo and once the event is parsed to json, just send it using the streaming API. Something like:

```
client = bigquery.Client()
try:
        json_data = snowplow_analytics_sdk.event_transformer.transform(event)
        rows_to_insert = [
                json_data
        ]

        errors = client.insert_rows(table, rows_to_insert)

        if len(errors) > 0:
            for error in errors:
                print("error: " + str(error))
        else:
            print("Row inserted")

        kafka_producer.poll(0)
        kafka_producer.produce('snowplow_json_event', json.dumps(json_data).encode('utf-8'))
```

You'll need a table with the proper columns in BigQuery. I'll be adding an example soon in this repo.

**Comment by lihan** [comment]
@pacuna Thank you. My questions are more around how to get schema setup in BigQuery. If I added custom events to JS tracker, how to reflect that change too.

**Comment by pacuna** [comment]
@lihan Oh I see. We haven't automated the schema management yet. We created the atomic event schema in bigquery and for the unstruct events we added the necessary rows. We use the one-table approach which has worked fine so far.
So basically every time we add a new custom event, we also add the new columns to the schema and adapt the loader script to extract the columns from the snowplow event and send the row to bigquery.
